src/packages.ts

```diff
@@ -21,7 +21,6 @@ export interface Package {
     fallbackUrl?: string;
     installPath?: string;
     platforms: string[];
-    runtimeIds: string[];
```

Comment:
Its certainly theoretically possible, but I felt like it was a lot of effectively dead code to keep around when we aren't sure how we would use it. I felt like it was better to remove it, and if we want it back we can go into the history and undo this change.